### PR TITLE
hardening/365_fix_default_parameters

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue #365 
* 100% unit test passed: 
```
Results : Tests run: 88, Failures: 0, Errors: 0, Skipped: 0
```
* e2e (unofficial) test passed: 
```
CKAN
time=2016-02-29T12:11:58.993CET | lvl=DEBUG | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionCKANSink[177] : [ckan-sink] CKAN persistence backend created
time=2016-02-29T12:11:58.993CET | lvl=INFO | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionSink[236] : [ckan-sink] Startup completed

MYSQL
time=2016-02-29T12:30:45.161CET | lvl=DEBUG | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionMySQLSink[154] : [mysql-sink] MySQL persistence backend created
time=2016-02-29T12:30:45.161CET | lvl=INFO | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionSink[236] : [mysql-sink] Startup completed

POSTGRESQL
time=2016-02-29T12:36:39.567CET | lvl=DEBUG | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionPostgreSQLSink[167] : [postgresql-sink] PostgreSQL persistence backend created
time=2016-02-29T12:36:39.588CET | lvl=INFO | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionPostgreSQLSink[176] : [postgresql-sink] Startup completed

MONGO
time=2016-02-29T12:51:55.808CET | lvl=DEBUG | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionMongoBaseSink[141] : [mongo-sink] MongoDB persistence backend created
time=2016-02-29T12:51:55.809CET | lvl=INFO | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionSink[236] : [mongo-sink] Startup completed

STH
time=2016-02-29T12:53:46.140CET | lvl=DEBUG | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionMongoBaseSink[141] : [sth-sink] MongoDB persistence backend created
time=2016-02-29T12:53:46.141CET | lvl=INFO | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionSink[236] : [sth-sink] Startup completed

KAFKA
time=2016-02-29T12:57:42.798CET | lvl=DEBUG | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionKafkaSink[131] : [kafka-sink] Kafka persistence backend (KafkaProducer) created
time=2016-02-29T12:57:42.971CET | lvl=INFO | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionSink[236] : [kafka-sink] Startup completed

DYNAMODB
time=2016-02-29T13:00:56.669CET | lvl=DEBUG | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionMySQLSink[154] : [dynamodb-sink] MySQL persistence backend created
time=2016-02-29T13:00:56.669CET | lvl=INFO | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionSink[236] : [dynamodb-sink] Startup completed

HDFS
time=2016-02-29T13:44:41.131CET | lvl=DEBUG | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionHDFSSink[518] : [hdfs-sink] HDFS persistence backend created
time=2016-02-29T13:44:41.133CET | lvl=INFO | trans= | srv= | subsrv= | function=start | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionSink[236] : [hdfs-sink] Startup completed
```
* PR derived from a re-opened issue: no CHANGES_NEXT_RELEASE update is required
* Assignee: @frbattid

### CODE REVIEW

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionPostgreSQLSink.java [hunk 1]
```diff
@@ -141,7 +141,7 @@ public void configure(Context context) {
         postgresqlPassword = context.getString("postgresql_password", "unknown");
```
Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionPostgreSQLSink.java]: I've seen this "unknown" default password... I guess all the default passwrods are empty in the templates, right?

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java [hunk 2]
```diff
@@ -130,7 +130,7 @@ public void configure(Context context) {
         mysqlPassword = context.getString("mysql_password", "unknown");
```
Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java]: I've seen this "unknown" default password... I guess all the default passwrods are empty in the templates, right?

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionPostgreSQLSink.java (hunk 1 above)
Comment (pcoello25) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionPostgreSQLSink.java]: Yes, you're right. Same issue found in `OrionMySQLSink`. 

Fixed both in a272748

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java (hunk 2 above)
Comment (pcoello25) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java]: Yes, you're right. Fixed in a272748

**Comment by frbattid** [comment]
LGTM

**Pull request merged**